community/graphdb-api/src/main/java/org/neo4j/graphdb/schema/ConstraintDefinition.java

```diff
@@ -28,7 +28,7 @@
  * <b>Note:</b> This interface is going to be changed/removed in next major release to better cope with node and
```

Comment:
please remove that or change, interface should be or @Deprecated or not, that javadoc part is just super confusing and wrong